repolinter-rulesets/nr1-lib-deprecations.yml

```
@@ -0,0 +1,116 @@
+# Schema: https://raw.githubusercontent.com/newrelic-forks/repolinter/master/rulesets/schema.json
+version: 2
+axioms: {}
+rules:
+  nr1-lib-deprecation-dropdown-label:
+    level: error
+    rule:
+      type: file-not-contents
+      nocase: true
+      options:
+        globsAll:
+        - "**/*.@(js|ts|jsx|tsx)"
```

Since you're searching for every javascript file, you'll probably also want to exclude `node_modules`:

```suggestion
        - "**/*.@(js|ts|jsx|tsx)"
        - "!node_modules/**"
```

---

ooh, good catch!

---

Good call. Thx.

---

😄